block/block.go

```diff
@@ -12,7 +12,7 @@ type (
 	base struct {
 		ConsensusData uint64
 		Index         uint32
-		Timestamp     uint32
+		Timestamp     uint64
```

Comment:
I think we can get away without changing this `Timestamp`, let it be uint32, it doesn't really affect dbft protocol.